src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/repository/BackupRepository.java

```diff
@@ -90,6 +87,9 @@ public OffsetDateTime getCreatedAt(String filename) {
      * @param filename
      */
     public void deleteBackupFile(String filename) {
+        if (!FormatUtils.isValidBackupFilename(filename)) {
+            throw new RuntimeException("invalid backup filename");
```

Comment:
SonarQube requires `"invalid backup filename"` to be in a constant because it occurs at least 3 times in this class.

Reply:
Fixed